kpops/pipeline.py

```diff
@@ -83,6 +87,71 @@ def to_yaml(self) -> str:
             self.model_dump(mode="json", by_alias=True, exclude_none=True)["components"]
         )
 
+    def build_execution_graph_from(
+        self,
+        components: list[PipelineComponent],
+        reverse: bool,
+        runner: Callable[[PipelineComponent], Coroutine],
+    ) -> Awaitable:
+        sub_graph_nodes = self.__get_graph_nodes(components)
+
+        async def run_parallel_tasks(tasks: list[Coroutine]) -> None:
```

Comment:
tasks seems like an inaccurate parameter name

Reply:
I replaced tasks for deployments